htmlreport/cppcheck-htmlreport

```diff
@@ -422,6 +422,7 @@ def git_blame(errors, path, file, blame_options):
     full_path = os.path.join(path, file)
     path, filename = os.path.split(full_path)
 
+    cwd = os.getcwd()
```

Comment:
@danmar This was introduced in c48fc9ef89bbca080e08b0535eba7cfe4ae1db31. If possible you should avoid using `chdir()`.

I will replace this with the proper usage of the `cwd` parameter in the `subprocess.check_output()` in a follow-up PR.

Reply:
yeah I agree `chdir()` should not be used if it can be avoided. :-(

Reply:
Most of them can actually: #6510.